integrationtest/backend-tck/src/main/java/org/hibernate/search/v6poc/integrationtest/backend/tck/util/TckConfiguration.java

```diff
@@ -42,11 +46,25 @@ private TckConfiguration() {
 		catch (IOException e) {
 			throw new IllegalStateException( "Error loading TCK properties file: " + PROPERTIES_PATH );
 		}
+
+		addTimestampToLuceneRootDirectory( properties );
```

Comment:
I really think we should instead replace occurrences of the `#{tck.startup.timestamp}` string with the actual timestamp in every property. I've tried hard to not tie the TCK to a particular backend implementation and I'd really like it to stay that way...
Note that while we're at it, we could also replace occurrences of `#{tck.test.id}` and instantiate the TckConfiguration once per test. It would allow us to end the practice of using the test ID in index names, because we could just inject the test ID in the Lucene base directory path.